sdk/purview/purview-workflow-rest/swagger/README.md

```diff
@@ -0,0 +1,24 @@
+# Azure Purview Workflow TypeScript Protocol Layer
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: "@azure-rest/purview-workflow"
+title: Purview Workflow
+description: Purview Workflow Client
+generate-metadata: true
+generate-test: true
+generate-sample: true
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/purviewWorkflow.json
+package-version: 1.0.0-beta.1
+rest-level-client: true
+add-credentials: true
+credential-scopes: "https://purview.azure.net/.default"
+use-extension:
+  "@autorest/typescript": "6.0.0-rc.3"
```

Comment:
Can you use 6.0.0-rc.8 https://www.npmjs.com/package/@autorest/typescript/v/6.0.0-rc.8 to regenerate this library ?

Reply:
Fixed, thanks